dask_cuda/proxify_host_file.py

```diff
@@ -142,13 +143,26 @@ class ProxifyHostFile(MutableMapping):
     ----------
     device_memory_limit: int
         Number of bytes of CUDA device memory used before spilling to host.
+    compatibility_mode: bool
+        Enables compatibility-mode, which means that items are un-proxified before
+        retrieval. This makes it possible to get some of the JIT-unspill benefits
+        without having to be ProxyObject compatible. In order to still allow specific
+        ProxyObjects, set the `mark_as_explicit_proxies=True` when proxifying with
+        `proxify_device_objects()`. If None, the "jit-unspill-compatibility-mode"
+        config value are used, which defaults to False.
     """
 
-    def __init__(self, device_memory_limit: int):
+    def __init__(self, device_memory_limit: int, compatibility_mode: bool = None):
```

Comment:
Is this allowed, to define the type as `bool` and assign `None` to it?

Reply:
I am not totally sure, but this is allowed by mypy by default: https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-optional